projects/core/bindaas-psuedo-sts/src/main/java/edu/emory/cci/bindaas/pseudosts/impl/PseduoSecurityTokenServiceImpl.java

```diff
@@ -3,12 +3,12 @@
 import java.util.Collection;
 import java.util.Dictionary;
 import java.util.Hashtable;
-
 import javax.ws.rs.GET;
 import javax.ws.rs.HeaderParam;
 import javax.ws.rs.Path;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.Response;
+import javax.xml.*;
```

Comment:
Looks to me like an unnecessary import. Pls remove and pls remove the file from the commit.